microprofile.h

```diff
@@ -1577,7 +1577,7 @@ uint64_t MicroProfileAllocateLabel(const char* pName)
 		}
 	}
 
-	uint32_t nLen = strlen(pName);
+	uint32_t nLen = (uint32_t)strlen(pName);

```

Comment:
It's better to change variable type to size_t here (assuming fetch_add does not complain - it should not...)